src/aiortc/rtcrtpparameters.py

```diff
@@ -56,116 +55,114 @@ def __str__(self):
         return s
 
 
-@attr.s
+@dataclass
 class RTCRtpRtxParameters:
-    ssrc: int = attr.ib()
+    ssrc: int
 
 
-@attr.s
+@dataclass
 class RTCRtpCodingParameters:
-    ssrc: int = attr.ib()
-    payloadType: int = attr.ib()
-    rtx: RTCRtpRtxParameters = attr.ib(default=None)
+    ssrc: int
+    payloadType: int
+    rtx: RTCRtpRtxParameters = None
```

Comment:
Optional[RTCRtpRtxParameters] ?

Reply:
Set type to `Optional[RTCRtpRtxParameters]`.
We are using `strict_optional = False` configuration in `setup.cfg` so mypy doesn't check for None type cases.